src/components/profileDropdown.tsx

```diff
@@ -30,7 +30,7 @@ function ProfileDropdown({
                 <div className="flex flex-col w-full gap-[5px] cursor-pointer">
                   <Link
                     onClick={handleShowProfileDropdown}
-                    to="/dashboard/profile"
+                    to="update-profile"
```

Comment:
Why did you change redirection link?

Reply:
Thanks, I solved that path to its original  path, I was using this path to reach update profile page,because there wasn't profile page to reach update-profile page , but thanks to you. For name files using camel I have done so for all files I have created, may those aren't mine. Thanks for reviews